resources/views/component/content/forum/list.blade.php

```diff
@@ -6,6 +6,7 @@
 
     @include('component.content.forum.list', [
         'modifiers' => $modifiers,
+        'container' => 'both',
```

Comment:
No component should never ever know about containers they are wrapped in. If possible, do not introduce container parameter, find a better abstraction with @apani even when it means more wrappers.